cirq-core/cirq/ops/phased_x_z_gate_test.py

```diff
@@ -313,3 +313,39 @@ def test_str_diagram() -> None:
 0: ───PhXZ(a=0.125,x=0.5,z=0.25)───
     """,
     )
+
+
+@pytest.mark.parametrize(
+    'clifford_gate',
+    [g.to_phased_xz_gate() for g in cirq.SingleQubitCliffordGate.all_single_qubit_cliffords],
+)
+def test_has_stabilizer_effect_true_for_cliffords(clifford_gate):
```

Comment:
Please add type annotations here and in new tests below -

```suggestion
def test_has_stabilizer_effect_true_for_cliffords(clifford_gate: cirq.PhasedXZGate) -> None:
```